readme.adoc

```diff
@@ -1,66 +1,63 @@
 == About this book
 
 _Bitcoin Development Philosophy_ is a guide for new Bitcoin developers
-who already understand the basics of, for example, proof of work, block
-building, and the transaction life cycle, who want to level-up and
-gain a deeper understanding of Bitcoin's design tradeoffs and
+who already understand the basics of concepts and processes such as Proof-of-Work, block
+building, and the transaction life cycle, and who want to level up by
+gaining a deeper understanding of Bitcoin's design trade-offs and
 philosophy. It should help new developers absorb the most important
-lessons of a decade of bitcoin development and discussion, and provide
+lessons of over a decade of Bitcoin development and discussion, and provide
 a context for evaluating new ideas (good ones and bad ones!).
 
 Table of contents overview:
 
 :oldstyle: {xrefstyle}
 :xrefstyle: full
-<<decentralization>>::
-<<trustlessness>>::
-<<privacy>>::
-<<finitesupply>>::
-<<upgrading>>::
-<<adversarialthinking>>::
-<<opensource>>::
-<<scaling>>::
-<<whenshithitsthefan>>::
-<<appendixdiscussion>>::
-<<appendixbuild>>:: {empty}
+<<Decentralization>>::
+<<Trustlessness>>::
+<<Privacy>>::
+<<Finitesupply>>::
+<<Upgrading>>::
+<<Adversarial thinking>>::
+<<Open source>>::
+<<Scaling>>::
+<<When shit hits the fan>>::
+<<Appendix discussion>>::
+<<Appendix build>>:: {empty}
 
 :xrefstyle: {oldstyle}
 
 Your feedback and contributions are most welcome! Instructions for
-building and contributing can be found in <<appendixbuild>>.
+building and contributing can be found in <<Appendix build>>.
```

Comment:
also shouldn't do this, same as: https://github.com/bitcoin-dev-philosophy/btcphilosophy/pull/58#discussion_r1072270158